src-tests/test-rpc.lisp

```diff
@@ -270,3 +270,94 @@
              (is (search "OOF-FIND-ME-ON-THE-STACK" (get-output-stream-string error-stream))))
         ;; kill the server thread
         (kill-thread-slowly server-thread)))))
+
+
+;; Canonical test keys from the the ZMQ curve docs
+;; http://api.zeromq.org/4-3:zmq-curve#toc4
+(defvar *server-public-key-z85* "rq:rM>}U?@Lns47E1%kR.o@n%FcmmsL/@{H8]yf7")
+(defvar *server-secret-key-z85* "JTKVSB%%)wK0E.X)V>+}o?pNmC{O&4W4b!Ni{Lh6")
+(defvar *client-public-key-z85* "Yne@$w-vo<fVvi]a<NY6T1ed:M$fCG*[IaLV{hID")
+(defvar *client-secret-key-z85* "D:)Q[IlAW!ahhC2ac:9*A}h:p?([4%wOTJ%JR%cs")
```

Comment:
`:)`

Reply:
some github bot is going to warn us that we checked in a secret key and prevent merging to master